lib/packer.ts

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable complexity */
 
 import assert from 'assert';
-import fs from 'fs-extra';
-import path from 'path';
+import * as fs from 'fs-extra';
+import * as path from 'path';
```

Comment:
This does not fit with other uses of `fs-extra` and `path`. 

and unnecessary?